stdlib/ballerina-mime/src/main/ballerina/mime/natives.bal

```diff
@@ -88,7 +88,7 @@ public type ContentDisposition object {
 
     documentation {
         Convert the ContentDisposition type to a string suitable for use as the value of a corresponding MIME header.
-        R{{}} Return the ContentDisposition object's content as a string
+        R{{}} Return a `ContentDisposition` object's content as a string
```

Comment:
Shall we rephrase this? Since this a function of an object, it won't be returning the string value of just any object. Only the object it's bound to